Cargo.toml

```diff
@@ -117,12 +117,6 @@ path = "parity/lib.rs"
 path = "parity/main.rs"
 name = "parity"
 
-[profile.dev]
-
-[profile.test]
-lto = false
-opt-level = 3 # makes tests slower to compile, but faster to run
-
```

Comment:
This might work: RUSTFLAGS="-C opt-level=3 -C overflow-checks=off -C incremental=false -C debuginfo=2" time cargo test -v
##

These are the settings for release mode I think:

# The release profile, used for `cargo build --release` (and the dependencies
# for `cargo test --release`, including the local library or binary).
[profile.release]
opt-level = 3
debug = false
rpath = false
lto = false
debug-assertions = false
codegen-units = 16
panic = 'unwind'
incremental = false
overflow-checks = false